summary-release-notes.md

```diff
@@ -0,0 +1,133 @@
+
+    - The discriminator `propertyName` can now be an optional field.
+    - New field `defaultMapping` to indicate which schema to use if the `propertyName` is not set, or if the value is unrecognized.
+
+- Templates with formal syntax
+
+    - The specification now includes **ABNF** (Augmented Backus–Naur Form) for path templating, server variables, and runtime expressions in the Links object.
+
+- Flexible metadata fields in the Response object
+
+    - `description` field for responses are now optional.
+    - Additional `summary` field for responses.
+
+- Additional updates
+
+    - Non-Schema examples no longer "override" Schema examples; tools are free to use the most appropriate example for any given use case.
+    - A new key `mediaTypes` is supported under `components` to support re-use of Media Type Objects.
+
+- Updates to referenced standards
+
+    - Update to <https://www.ietf.org/archive/id/draft-bhutton-json-schema-01.html> of JSON Schema Specification.
+    - Update to <https://www.ietf.org/archive/id/draft-bhutton-json-schema-validation-01.html> of JSON Schema Validation Specification.
+    - Use [RFC8529](https://tools.ietf.org/html/rfc8259) for JSON.
+    - Use [RFC9110](https://tools.ietf.org/html/rfc9110) for HTTP.
+
+- Editorial changes
+
+    - Extensive additions around media types, encoding, sequential media types, SSE examples, working with binary data.
+    - Streamlined to YAML examples (unless something specific to another format) to try to make it easier to follow.
+    - Better explanation and examples for using Encoding and Serialization, and a note not to apply percent-encoding to headers.
+    - Clarify that Request Body Objects need to specify at least one media type to be meaningful.
+    - How to more clearly indicate that responses will not have a body.
+    - Explanation and examples of headers including `Link` and `Set-Cookie`.
+    - Detailed explanation of percent-encoding and -decoding, updated examples and references to match, including the Style Examples table.
+    - Extensive additional notes on parsing and serializing JSON and non-JSON data formats.
+
+## 3.1 Updates
+
+Version 3.1.2 has no material changes but does contain editorial fixes, additional examples, and clarifications.
+
+- Clarify that `$ref` in a Schema Object is a JSONSchema `$ref` keyword.
+- Detailed explanation of percent-encoding and -decoding, updated examples and references to match, including the Style Examples table.
+- Better explanation and examples for using Encoding and Serialization, and a note not to apply percent-encoding to headers.
+- Clarify that Request Body Objects need to specify at least one media type to be meaningful.
+- How to more clearly indicate that no response will have a body.
+- How to handle `null` in XML as an advisory note; since the functionality cannot be changed it is implementation-defined for 3.1 tooling.
+- Clarify that the root schema of an XML object should use the component name.
```

Comment:
```suggestion
- Clarify that the root schema of an XML Object should use the component name.
```

Reply:
I think in this instance it's an object in the XML rather than a proper noun that we should Frighteningly Capitalise.